contents/swift/welcome-to-swift.md

```diff
@@ -218,9 +261,9 @@ barcode2.printCode() // prints "QR code: bar."
 
 Also, enums with associated values is not supported in languages such as [Java](https://stackoverflow.com/questions/30044334/how-can-i-create-a-java-enum-with-associated-values-like-swift-enum), and using a workaround to implement enums with associated values results in code verbosity. Take a look at [Swift's documentation on Enums](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Enumerations.html) for more information about enums.
 
-## Protocol Oriented Programming
+### Benefit: Protocol Oriented Programming
 
-The heart of Swift is Protocol Oriented Programming (POP) which is about abstraction and simplicity. POP helps to solve the [bloat that is sometimes caused by Object Oriented Programming (OOP)](http://blogs.perl.org/users/sid_burn/2014/03/inheritance-is-bad-code-reuse-part-1.html). If you ever find yourself having to inherit from multiple classes, you probably should consider using protocols instead.
+The heart of Swift is Protocol Oriented Programming (POP) which is about abstraction and simplicity. POP helps to solve the [bloat that is sometimes caused by Object Oriented Programming (OOP)](http://blogs.perl.org/users/sid_burn/2014/03/inheritance-is-bad-code-reuse-part-1.html). POP uses a kind of composition instead of inheritance for implementation. Therefore, if you ever find yourself having to inherit from multiple classes, you probably should consider using protocols instead.
```

Comment:
> POP uses a kind of composition instead of inheritance for implementation.

Possible to rephrase this or expand a little bit? After reading this sentence, my first thought is "what kind?"